packages/agoric-cli/agoric-wallet-build/asset-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
   "files": {
-    "main.js": "./static/js/main.8b821acd.chunk.js",
-    "main.js.map": "./static/js/main.8b821acd.chunk.js.map",
+    "main.js": "./static/js/main.2de9e919.chunk.js",
+    "main.js.map": "./static/js/main.2de9e919.chunk.js.map",
     "runtime-main.js": "./static/js/runtime-main.6d19b3b8.js",
     "runtime-main.js.map": "./static/js/runtime-main.6d19b3b8.js.map",
     "static/js/2.8b396fe9.chunk.js": "./static/js/2.8b396fe9.chunk.js",
     "static/js/2.8b396fe9.chunk.js.map": "./static/js/2.8b396fe9.chunk.js.map",
     "index.html": "./index.html",
-    "precache-manifest.dc3db1e8ecb4d6dc99cf7c3550f46d0c.js": "./precache-manifest.dc3db1e8ecb4d6dc99cf7c3550f46d0c.js",
+    "precache-manifest.81ac76d1232f29be088449d54b9983b8.js": "./precache-manifest.81ac76d1232f29be088449d54b9983b8.js",
```

Comment:
Are we checking generated files into git? Is there a better way?

Reply:
The `wallet-frontend` needs to be built and the golden copy in `agoric-cli` updated as a different step.  I don't really want to force everybody to install React and its entire ecosystem just to get a wallet.

There may be a way to separate `@agoric/wallet-frontend` into its own package and publish the build on NPM so that `agoric-cli` can use it.  I'll open an issue.